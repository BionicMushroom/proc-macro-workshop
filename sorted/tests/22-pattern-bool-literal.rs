#[allow(clippy::match_bool)]
#[sorted::check]
fn main() {
    let x = false;

    #[sorted]
    match x {
        true => println!("true"),
        false => println!("false"),
    }
}
