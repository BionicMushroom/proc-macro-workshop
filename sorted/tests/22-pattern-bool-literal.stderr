error: false should sort before true
 --> tests/22-pattern-bool-literal.rs:9:9
  |
9 |         false => println!("false"),
  |         ^^^^^
